### ISSUE TITLE
A11y_VisualStudioCodeClient_Version Control_ Screen reader:NVDA/Narrator is not announcing comments like “You last week add first commit” in visual code editor

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#7203]
--- GitHub Tags:
-A11yMAS; -A11yTCS; -A11ySev2; -DesktopApp; -Win32; -Visual Studio Code Client; -BM-VisualStudioCodeClient-Win32-Aug2022; -WCAG1.3.1; -NVDA; -ScreenReader; -Narrator;

--- Environment and OS details:
Visual Studio Code Version: 1.70.2 (user setup)
Windows 11 Pro version 21H2 (OS Build 22000.856)

--- Repro Steps:

1.	Open Visual studio code editor and create branch in new folder by using https://code.visualstudio.com/docs/editor/versioncontrol
2.	Open a file in that folder
3.	Navigate in file using keyboard
4.	Verify that screen reader announcing the comments like “You last week add first commit” 
Or not 

--- Actual Result:
NVDA/Narrator is not announcing comments like “You last week add first commit” in visual code editor

--- Expected Result:
NVDA/Narrator should announce comments like “You last week add first commit” in visual code editor

--- Attachment:
![A11y-VisualStudioCodeClient-Vers](https://user-images.githubusercontent.com/100189407/187403263-0075817f-7d81-4407-9c09-2d54e8079b20.gif)
![A11y-VisualStudioCodeClient-Vers (1)](https://user-images.githubusercontent.com/100189407/187406400-b6b36f75-7d3c-48fc-bf2c-a52531e54630.gif)
